src/material/datepicker/multi-year-view.html

```diff
@@ -1,8 +1,9 @@
-<table class="mat-calendar-table" role="presentation">
-  <thead class="mat-calendar-table-header">
+<table class="mat-calendar-table" role="grid">
+  <thead aria-hidden="true" class="mat-calendar-table-header">
     <tr><th class="mat-calendar-table-header-divider" colspan="4"></th></tr>
   </thead>
   <tbody mat-calendar-body
+         role="rowgroup"
```

Comment:
You forgot this one, @jelbourn :)

Reply:
Done 👍

Reply:
It seems like the last commit reverted everything to the previous state (where all roles were setted).

Reply:
Oh, good catch. I ssh'ed in my terminal and then pushed without thinking from an outdated branch.